ee/src/main/java/org/jboss/as/ee/subsystem/ManagedScheduledExecutorServiceResourceDefinition.java

```diff
@@ -84,6 +86,15 @@
                     .setValidator(new StringLengthValidator(0, true))
                     .setFlags(AttributeAccess.Flag.RESTART_RESOURCE_SERVICES)
                     .setCapabilityReference(ManagedThreadFactoryResourceDefinition.CAPABILITY.getName(), CAPABILITY)
+                    .setDeprecated(EESubsystemModel.Version.v5_0_0)
+                    .build();
+
+    public static final SimpleAttributeDefinition THREAD_PRIORITY_AD =
+            new SimpleAttributeDefinitionBuilder(THREAD_PRIORITY, ModelType.INT, true)
```

Comment:
This and THREAD_FACTORY_AD should both use setAlternatives to declare that each is an alternative to the other.